File: fib-simple/scala/fib.scala
@main def main(args: String*) =
    val n = args(0)
    printf("fib(%s) = ",n)
    println(fib(n.toInt))
    
def fib(n:Long) : Long=
  if n < 2 then
    n
  else
    fib(n-1)+fib(n-2)

File: fib-simple/golang/fib.go
package main

import (
	"fmt"
)

func fib(n int) int {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	fmt.Println("fib(42)=", fib(42))
}
